ui/ui-components/pages/model/[modelId]/overview.tsx

```diff
@@ -50,8 +50,6 @@ const Page: NextPage<Props> = ({
     [properties]
   );
 
-  if (!model) return <Alert variant="warning">Model not found</Alert>;
```

Comment:
This is no longer needed because we now fall into the hydration error when requests in getServerSideProps fail